src/main/java/de/caritas/cob/userservice/api/helper/LogstashAppender.java

```diff
@@ -91,4 +129,12 @@ private void logToStandardError(String msg) {
   public void setEncoder(Encoder<T> encoder) {
     this.encoder = encoder;
   }
+
+  public void setLogBufferSize(int logBufferSize) {
```

Comment:
Are the setter methods only for unit tests? Could you please remove them from production code and use REflectionTestUtils.setField() instead?

Reply:
they are not just for unit tests, as we want to be able to specify them in the logstash-spring.xml